API/Helpers/Builders/MangaFileBuilder.cs

```diff
@@ -60,4 +60,14 @@ public MangaFileBuilder WithId(int id)
         _mangaFile.Id = Math.Max(id, 0);
         return this;
     }
+
+    public MangaFileBuilder WithHash()
+    {
+        if (_mangaFile.Format == MangaFormat.Epub)
```

Comment:
Reminder to self, inversion of control this (aka check if not epub and return this)